record-builder-processor/src/main/java/io/soabase/recordbuilder/processor/RecordBuilderMetaDataLoader.java

```diff
@@ -17,23 +17,34 @@
 
 import io.soabase.recordbuilder.core.RecordBuilderMetaData;
 
+import java.lang.reflect.InvocationTargetException;
+import java.util.Map;
 import java.util.function.Consumer;
 
+import javax.annotation.processing.ProcessingEnvironment;
+
 class RecordBuilderMetaDataLoader {
     private final RecordBuilderMetaData metaData;
 
-    RecordBuilderMetaDataLoader(String metaDataClassName, Consumer<String> logger) {
-        RecordBuilderMetaData localMetaData = null;
+    RecordBuilderMetaDataLoader(ProcessingEnvironment processingEnv, Consumer<String> logger) {
+        Map<String, String> options = processingEnv.getOptions();
+        String metaDataClassName = options.get(RecordBuilderMetaData.JAVAC_OPTION_NAME);
```

Comment:
The old pure interface based configuration implementation is untouched and should work if this PR is merged. If no meta-data class name is provided by the developer, we fall back to the new -A option based configuration.